royalties-registry/contracts/RoyaltiesRegistry.sol

```diff
@@ -93,34 +95,55 @@ contract RoyaltiesRegistry is IRoyaltiesProvider, OwnableUpgradeable {
     }
 
     function royaltiesFromContract(address token, uint tokenId) internal view returns (LibPart.Part[] memory) {
+        (LibPart.Part[] memory royalties, bool ok) = royaltiesFromContractNative(token, tokenId);
```

Comment:
name "isNative" will have more convenient meaning instead of "ok"

Reply:
agree, use isNative